packages/bot-web-ui/src/pages/tutorials/constants.ts

```diff
@@ -422,7 +422,7 @@ export const faq_content: TFaqContent[] = [
             {
                 type: 'text',
                 content: localize(
-                    '<a href="https://www.youtube.com/watch?v=QdI5zCkO4Gk&t=203s" target="_blank">Watch this video</a> to learn how to build a trading bot on Deriv Bot. Also, <a href="https://deriv.com/academy/blog/posts/how-to-build-a-basic-trading-bot-with-dbot/" target="_blank">check out this blog post</a> on building a trading bot.'
+                    '<a href="https://www.youtube.com/watch?v=QdI5zCkO4Gk&t=203s" target="_blank">Watch this video</a> to learn how to build a trading bot on Deriv Bot. Also, <a href="https://blog.deriv.com/posts/how-to-build-a-basic-trading-bot-with-dbot/" target="_blank">check out this blog post</a> on building a trading bot.'
```

Comment:
```suggestion
                    '<a href="https://www.youtube.com/watch?v=QdI5zCkO4Gk&t=203s" rel="noopener noreferrer" target="_blank">Watch this video</a> to learn how to build a trading bot on Deriv Bot. Also, <a href="https://blog.deriv.com/posts/how-to-build-a-basic-trading-bot-with-dbot/" target="_blank" rel="noopener noreferrer" >check out this blog post</a> on building a trading bot.'
```

Best to add `rel="noopener noreferrer" ` if you are opening a link in a new tab: some references: https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/

Reply:
Sure @yashim-deriv Done